sdk/servicebus/Azure.Messaging.ServiceBus/perf/Infrastructure/ServiceBusPerfTestBase.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus.Tests;
+using Azure.Test.Perf;
+
+namespace Azure.Messaging.ServiceBus.Perf
+{
+    public abstract class ServiceBusPerfTestBase : PerfTest<SizeCountOptions>
+    {
+        protected ServiceBusScope.QueueScope QueueScope;
+        private readonly bool _useSessions;
+        protected ServiceBusClient Client { get; private set; }
+        protected ServiceBusSender Sender { get; private set; }
+        protected ServiceBusReceiver Receiver { get; private set; }
+        protected int SeededMessageCount { get; }
+
+        // until https://github.com/Azure/azure-sdk-for-net/issues/17248 is addressed we have to estimate how many
+        // messages may be needed
+        private const int s_operationCountEstimate = 1000;
```

Comment:
nit:  Don't we normally use `s_` only on static variables and just a leading capital on constants?

Reply:
hmm.. this was recommended by Rider, let me see what VS does.